listings/ch00-introduction/if_else/src/access_control.cairo

```diff
@@ -0,0 +1,23 @@
+use starknet::ContractAddress;
+
+#[contract]
+mod SimpleAccessControl {
+    use starknet::get_caller_address;
+
+    struct Storage {
+        owner: starknet::ContractAddress, 
```

Comment:
by convention, it's better to import it in the contract with `use starknet::{get_caller_address,ContractAddress}` and just use `ContractAddress`